server.js

```diff
@@ -7888,6 +7888,73 @@ camp.route(/^\/vaadin-directory\/(star|status|rating|rc|rating-count|v|version|r
 
 }));
 
+// Bugzilla bug integration
+camp.route(/^\/bugzilla\/(\d+)\.(svg|png|gif|jpg|json)$/,
+cache(function (data, match, sendBadge, request) {
+  var bugNumber = match[1];  // eg, 1436739
+  var format = match[2];
+  var options = {
+    method: 'GET',
+    json: true,
+    uri: 'https://bugzilla.mozilla.org/rest/bug/' + bugNumber
+  };
+  var badgeData = getBadgeData('Bug ' + bugNumber, data);
```

Comment:
could you also change this to be lower case
eg: ![](https://img.shields.io/:bug%20996038-fixed-brightgreen.svg)